src/Chapter3.hs

```diff
@@ -459,6 +485,19 @@ and provide more flexibility when working with data types.
 Create a simple enumeration for the meal types (e.g. breakfast). The one who
 comes up with the most number of names wins the challenge. Use your creativity!
 -}
+data Meal
+  = PreSwimmingLesson5AMSnack
+  | LittleBreakfast
+  | Breakfast
+  | MidMorningSnack
+  | Lunch
+  | ExtendedLunch
+  | MidAfternoonSnack
+  | DinnerAppetizer
+  | DinnerSoup
+  | DinnerEntree
+  | DinnerDessert
+  | MidnightSnack
```

Comment:
Very nice menu! :heart_eyes: I see a candidate for winning :1st_place_medal:

Reply:
Thanks, I tried my best!